src/components/mgt-person/mgt-person.ts

```diff
@@ -592,7 +592,7 @@ export class MgtPerson extends MgtTemplatedComponent {
   ): TemplateResult {
     const title =
       this.personDetails && this.personCardInteraction === PersonCardInteraction.none
-        ? this.personDetails.displayName
+        ? this.personDetails.displayName || this.personDetails.mail || ''
```

Comment:
Use `getEmailFromGraphEntity` instead of `.mail` and avoid adding the mail property o IDynamicPerson below.

Reply:
Good idea! Done